docs/rfcs/0013-rfc-process-updates.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: page
+title: RFC process updates
+permalink: /docs/rfcs/0013-rfc-process-updates
+---
+
+* RFC PR: /* TODO */
+* Start date: 2024-02-20 
+* End date: 2024-03-22 
+* Stakeholders: @jonalmeida, @boek
+
+## Summary
+
+Clarify requirements for RFCs by introducing a README and a template.
+
+## Motivation
+
+- The scale of the Firefox Android team has grown substantially since the RFC process was first introduced. 
+- More external teams are using and contributing to Android Components. 
+- Approval or rejection of a proposal has required periods of time that can slow implementation.
+- Stakeholders have either been implicit by sharing RFCs in various communication channels, or an author must be expected to correctly select reviewers that lack explicitly documented ownership of affected areas.
+
+## Guide-level explanation
+
+This proposal suggests improving the RFC process by introducing the following:
+
+- an RFC template
+- a guide for when RFCs are appropriate and when they are not needed in the form of a README
+- explicit stakeholders for RFCs
+- an initial deadline for each RFC
+- a new RFC section, "artifacts" which contain links to any additional artifacts introduced alongside the RFC
+- updating our CODEOWNERS file to better facilitate ownership for new Firefox Android "groups" and external teams
```

Comment:
I think using CODEOWNERS as part of the new process is great, but let's not couple updating CODEOWNERS as part of this RFC. With that we can also remove the explanation of why we're not actually updating the file with this change.

Reply:
Yeah, fair enough. I filed [this follow-up to handle CODEOWNERS](https://bugzilla.mozilla.org/show_bug.cgi?id=1881373) and will include it in the artifacts for this RFC